docs/quickstart.rst

```diff
@@ -0,0 +1,163 @@
+Quickstart
+==========
+
+The easiest way to get started with ethPM.
+
+Installation
+------------
+
+.. code-block:: bash
+
+   brew update
+   brew upgrade
+   brew tap ethpm/ethpm-cli
+   brew install ethpm-cli
+
+Or in a fresh Python `virtual environment <https://uoa-eresearch.github.io/eresearch-cookbook/recipe/2014/11/26/python-virtual-env/>`__.
+
+.. code-block:: bash
+
+   pip install ethpm-cli
+
+`More installation details and options <installation.html#installation>`__
+
+Setting your environment
+------------------------
+
+Infura (required)
+~~~~~~~~~~~~~~~~~
+
+ethPM currently uses `Infura <https://infura.io/>`__ to talk to the blockchain, so you must provide an Infura API key to authenticate your requests. It's free and simple to sign up for a key. You can sign up for a key `here <https://infura.io/register>`__ and then use the following command to set your key as the environment variable ``WEB3_INFURA_PROJECT_ID``.
+
+.. code-block:: bash
+
+   export WEB3_INFURA_PROJECT_ID=abc123...xyz890
+
+
+Wallet account (optional)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Setting a wallet account authentication is necessary to use commands that sign a transaction (i.e. deploying an ethPM registry, releasing an ethPM package to a registry). ethPM uses  `eth-keyfile <https://github.com/ethereum/eth-keyfile>`__ to interact with your encrypted private keyfile and sign transactions. Follow the steps in the README to generate your keyfile, and then use the following command to link your keyfile to ethPM.
+
+.. code-block:: bash
+
+   ethpm auth --keyfile-path KEYFILE_PATH
+
+Test that your keyfile has been properly stored.
+
+.. code-block:: bash
+
+   ethpm auth
+   > Keyfile stored for address: 0x123abc....................
+
+You can now use your keyfile's password as the flag ``--keyfile-password`` for any ethPM command, and it will be used to automatically sign any transactions.
```

Comment:
Nit: change 'as the flag' -> 'with the flag' as it sounds like your password could be used as an argument by typing `--[my-password]`